src/components/StorybookMobileDeviceSimulator/StorybookMobileDeviceSimulator.js

```diff
@@ -26,6 +33,34 @@ const deviceNameToResolutionLookup = {
     width: '320px',
     height: '568px',
   },
+  [deviceNameLookup.samsungA30]: {
+    width: '412px',
+    height: '892px',
+  },
+  [deviceNameLookup.samsungA50]: {
+    width: '412px',
+    height: '892px',
+  },
+  [deviceNameLookup.huaweiMate20Lite]: {
+    width: '360px',
+    height: '780px',
+  },
+  [deviceNameLookup.small]: {
```

Comment:
Please sort these `small`, `medium`, `large`, and `extraLarge` devices to the top of the array (just after the `default` one), so that they appear first in the list in the Storybook Knob.